dist/switch/switch.css

```diff
@@ -64,13 +65,31 @@ input.switch__control:focus + span.switch__button {
 input.switch__control:focus:not(:focus-visible) + span.switch__button {
   outline: none;
 }
-span.switch__control[aria-checked="true"] + span.switch__button::after {
-  left: 19px;
+input.switch__control[disabled] + span.switch__button::after,
+span.switch__control[aria-disabled="true"] + span.switch__button::after {
+  border-color: var(--switch-border-color, var(--color-background-disabled));
+}
+input.switch__control:not([disabled]):hover + span.switch__button,
+span.switch__control:not([aria-disabled="true"]):hover + span.switch__button {
```

Comment:
We might need this for focus too. On the checked version the focus is hard to see.

Reply:
Good call. I'll check.